pyproject.toml

```diff
@@ -3,6 +3,7 @@ name = "secureli"
 version = "0.2.14"
 description = "Secure Project Manager"
 authors = ["Tristan Leonard <tristanl@slalom.com>"]
```

Comment:
Do we want to change the author?

Reply:
I technically didn't author it, but I am ultimately responsible so let's switch it to me. caleb.tonn@slalom.com